### PR TITLE
Raise WasmUnsupportedError for ffmpeg usage on Lite

### PR DESCRIPTION
**Pull request by whitphx**
FFMpeg should be supported finally, but at this moment we set errors for easy debugging.

### CODE REVIEW

**Review by freddyaboulton — COMMENTED** [review]
I think the audio component uses ffmpeg under the hood in `save_audio_to_cache` functions. Playground link:

https://www.gradio.app/playground?demo=Hello_World&code=aW1wb3J0IGdyYWRpbyBhcyBncgoKZGVmIGdyZWV0KG5hbWUpOgogICAgcmV0dXJuIG5hbWUKCmRlbW8gPSBnci5JbnRlcmZhY2UoZm49Z3JlZXQsIGlucHV0cz0iYXVkaW8iLCBvdXRwdXRzPWdyLkF1ZGlvKGZvcm1hdD0ibXAzIikpCgppZiBfX25hbWVfXyA9PSAiX19tYWluX18iOgogICAgZGVtby5sYXVuY2goKQo=

**Comment by whitphx** [comment]
@freddyaboulton Correct! Thank you, I added it.

File: gradio/components/audio.py [hunk 1]
```diff
@@ -310,6 +310,10 @@ def postprocess(
 
     @staticmethod
     def _convert_to_adts(data: bytes):
+        if wasm_utils.IS_WASM:
```
Comment (freddyaboulton) [inline gradio/components/audio.py:313]: I would also raise if `needs_conversion` is `True` in `preprocess`. I am pretty sure that `pydub` is using ffmpeg under the hood in that case.

**Review by freddyaboulton — APPROVED** [review]
Thanks @whitphx !

**Pull request merged**